doc/cephadm/monitoring.rst

```diff
@@ -0,0 +1,70 @@
+Monitoring Stack with Cephadm
+=============================
+
+The Ceph dashboard makes use of prometheus, grafana, and related tools
+to store and visualize detailed metrics on cluster utilization and
+performance.  Ceph users have three options:
```

Comment:
I'm not entirely too sure what the style says, but I haven't seen any double spaces after a period in other reviews. Do we have a style guide for that kind of thing? Either way, I'd recommend removing it. It's redundant in online documentation.

Reply:
the whitespace spacing is handled by the browser and doesn't distinguish between (any number of) spaces, newlines, or tabs.

Reply:
Sure. I guess that's my point, no need to include the double space if it doesn't distinguish the spaces :) it just adds to the char count on the line.